.github/workflows/run_tests.yml

```diff
@@ -78,12 +78,13 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
-    - uses: actions/setup-python@v2
+    - uses: s-weigand/setup-conda@v1
     - uses: actions/setup-node@v1.4.2
     - name: Install dependencies
       run: |
         cd "${GITHUB_WORKSPACE}"
         pip install -r requirements.txt
+        conda install -c rdkit rdkit
```

Comment:
I don't understand the changes in this file. Are they conceptually related to the rest of the PR?

Reply:
Editor now requires RDKit to run (because RDKit is imported for use in validation functions of serve.py), so we had to instal it in the testing workflow. Conda's needed to install rdkit.